docs/pages/blog/2023-toolpad-beta-announcement.md

```diff
@@ -0,0 +1,81 @@
+
+## What are the main features available today?
+
+### Use MUI Core and MUI X Pro components
+
+MUI Core provides all modern components; a chosen few are currently available inside Toolpad, and we're adding more all the time. MUI X Pro (Data Grid and Date Picker) components are also available for free in Toolpad.
+
+### Bring your own components
+
+Toolpad supports bringing your [external components](https://mui.com/toolpad/concepts/custom-components/) to the project. Your past effort should not go to waste—we don't believe in a walled garden approach. In the future, we'll also support importing from Storybook and GitHub.
+
+### Bring your own back end
+
+Directly integrate with your Node.js back end and have your data available on the page without writing any REST endpoints or fetch logic.
+
+### Write custom functions in your IDE
+
+Toolpad works with your code editor and lets you write [custom functions](https://mui.com/toolpad/concepts/connecting-to-data/#custom-functions). These functions:
+
+- run on the back end.
+- return the data that is made available on the page without requiring you to set up any endpoints or data-fetching logic.
+- support arguments can be bound directly to page state.
+- can be integrated not only with any node module, but also with your back-end code, such as Prisma models or database connections.
+
+### Own your code
+
+Toolpad runs fully locally. All configuration is stored in local files which you can version-control, edit, and deploy any way you want.
+
+## How is Toolpad different from similar products on the market?
+
+Toolpad follows an open-core approach, meaning that the base version is open source. You can build an unlimited number of apps, and the basic service will be free forever. Among similar products like Retool, Appsmith, Budibase, Airplane, and Interval, only some are open source.
+
+Another major area of difference is that Toolpad is solely focused on developers as users. As much as we are low-code, we are equally code-friendly. Providing basic functionalities that a developer expects from an open-source tool is non-negotiable for us—we will always prioritize the developer's best interests over all else. With nearly a decade of history building developer tools, we know the long-term benefits of cultivating a close relationship with our community of users.
+
+Lastly, Toolpad is the only product that offers a drag-and-drop UI builder closely integrated with your favorite IDEs like VSCode. It's the best of both the worlds!
+
+## How can I use Toolpad?
+
+Toolpad is available as an NPM package—follow the [Installation guide](https://mui.com/toolpad/getting-started/installation/) in our docs to get started. You can learn more about Toolpad by visiting the [home page](https://mui.com/toolpad/).
```

Comment:
I saw these "—" a few times, was this on purpose?
```suggestion
Toolpad is available as an NPM package. Follow the [Installation guide](https://mui.com/toolpad/getting-started/installation/) in our docs to get started. You can learn more about Toolpad by visiting the [home page](https://mui.com/toolpad/).
```

Reply:
@Janpot that's an [em dash](https://www.thesaurus.com/e/grammar/em-dash/), AKA every editor's favorite punctuation mark. 😅 I'm probably responsible for all of them here. You can think of it as something like halfway between a period and a comma—it's meant to bridge two independent but related ideas, as I've attempted to demonstrate right here.